### PR TITLE
fix: Fix the command GenerateKeyPair

### PR DESCRIPTION
**Pull request by oharzallah-greenbull**
- Fix a typo in service decalaration
- Make algorithm as an argument because it was missing in the service definition and cannot be declared

### CODE REVIEW

**Comment by oharzallah-greenbull** [comment]
hi  @chalasr! can anyone check this PR please

File: src/Command/GenerateKeyPairCommand.php [hunk 1]
```diff
@@ -61,19 +59,20 @@ protected function configure(): void
         $this->addOption('dry-run', null, InputOption::VALUE_NONE, 'Do not update key files.');
         $this->addOption('skip-if-exists', null, InputOption::VALUE_NONE, 'Do not update key files if they already exist.');
         $this->addOption('overwrite', null, InputOption::VALUE_NONE, 'Overwrite key files if they already exist.');
+        $this->addArgument('algorithm', InputArgument::OPTIONAL, 'The algorithm code, possible values : RS256|RS384|RS512|HS256|HS384|HS512|ES256|ES384|ES512', 'RS256');
```
Comment (chalasr) [inline src/Command/GenerateKeyPairCommand.php:62]:
```suggestion
        $this->addArgument('algorithm', InputArgument::OPTIONAL, sprintf('The algorithm code, possible values : %s', implode(self::ACCEPTED_ALGORITHMS), 'RS256');
```

**Review by chalasr — APPROVED** [review]
👍 just one minor comment

File: src/Command/GenerateKeyPairCommand.php (hunk 1 above)
Comment (oharzallah-greenbull) [reply src/Command/GenerateKeyPairCommand.php]: Done !

**Comment by oharzallah-greenbull** [comment]
Do i have to change anything for the CI to work ?

**Comment by chalasr** [comment]
Nope, I'll take care of that given it's not related to your changes. Expect this to be merged next week. 
Thanks!

**Comment by chalasr** [comment]
Thank you @oharzallah-greenbull.

**Pull request merged**